api/README.md

```diff
@@ -81,8 +96,15 @@ Les tests pourront ensuite être exécutés avec ou sans docker-compose
 * Lancement des tests depuis la ligne de commande. Il est ainsi très simple d'accéder à `stdin`/`stdout` via le
   paramètre  `-s`, par exemple pour utiliser des breakpoints.
   ```shell
-  python -m pytest # Pour lancer tous les tests
-  python -m pytest tests/core/offers/test_api.py::CreateOfferTest::test_create_offer_from_scratch # Pour lancer un test en particulier
+  poetry run pytest # Pour lancer tous les tests
+  poetry run pytest tests/core/offers/test_api.py::CreateOfferTest::test_create_offer_from_scratch # Pour lancer un test en particulier
```

Comment:
Devoir préfixer toutes les commandes par `poetry` me semble fastidieux. Est-ce nécessaire, n'y a-t-il pas un moyen d'activer un virtualenv pour accéder aux commandes sans préfixe ?

Reply:
si, `poetry shell` @cepehang il y avait une raison ?

Reply:
J'avais mise la version plus complète de la commande `source $(poetry env info --path)/bin/activate` [expliquée dans la doc](https://python-poetry.org/docs/basic-usage#activating-the-virtual-environment) pour les gens qui ont plusieurs environnements virtuels par projet comme François, mais en effet le README mérite de parler de `poetry shell` en priorité